3rdparty/exported/t_cose/inc/t_cose/t_cose_common.h

```diff
@@ -13,6 +13,7 @@
 #define __T_COSE_COMMON_H__
 
 #include <stdint.h>
+#include <stdbool.h>
```

Comment:
Can you move the content of inc/t_cose back to t_cose/t_cose?

Reply:
As discussed, this may not be necessary now that it's inc/t_cose, let's see if the Daily passes and if it does we're fine.